sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ReifyTest.java

```diff
@@ -0,0 +1,213 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.sdk.transforms;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+import java.io.Serializable;
+
+import org.apache.beam.sdk.coders.StringUtf8Coder;
+import org.apache.beam.sdk.testing.PAssert;
+import org.apache.beam.sdk.testing.TestPipeline;
+import org.apache.beam.sdk.testing.TestStream;
+import org.apache.beam.sdk.testing.ValidatesRunner;
+import org.apache.beam.sdk.transforms.windowing.GlobalWindow;
+import org.apache.beam.sdk.transforms.windowing.PaneInfo;
+import org.apache.beam.sdk.values.KV;
+import org.apache.beam.sdk.values.PCollection;
+import org.apache.beam.sdk.values.TimestampedValue;
+import org.apache.beam.sdk.values.ValueInSingleWindow;
+import org.joda.time.Instant;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Utility transforms for reifying implicit context into explicit fields.
+ */
+@RunWith(JUnit4.class)
+public class ReifyTest implements Serializable {
+  public static final WithTimestamps<KV<String, Integer>>
+      TIMESTAMP_FROM_V = WithTimestamps.of(
+      new SerializableFunction<KV<String, Integer>, Instant>() {
+        @Override
+        public Instant apply(KV<String, Integer> input) {
+          return new Instant(input.getValue().longValue());
+        }
+      });
+  @Rule
+  public transient TestPipeline pipeline = TestPipeline.create();
+
+  /**
+   * Code coverage for package-private constructor.
```

Comment:
I'm not sure this one is worth the trouble.

Reply:
removed, reducing line coverage from 100% to 94% per IntelliJ